lib/pheromone/messaging/message_dispatcher.rb

```diff
@@ -38,6 +40,10 @@ def send_message_asynchronously
           Resque.enqueue(background_processor_klass, message_body)
         elsif background_processor.name == :sidekiq
           background_processor_klass.perform_async(message_body)
+        elsif background_processor.name == :custom
+          background_processor.custom_processor.call(background_processor_klass, message_body)
```

Comment:
does this need to be `constantize`d. `klass` in the setting above is a string so should we be consistent?